psi4/src/psi4/dfocc/ccsd_triples.cc

```diff
@@ -1656,7 +1657,9 @@ void DFOCC::ccsd_canonic_triples_grad2() {
     outfile->Printf("\tnthreads: %i \n", nthreads);
 
     // Find number of unique ijk combinations (i>=j>=k)
-    Nijk = naoccA * (naoccA + 1) * (naoccA + 2) / 6;
+    // Nijk = naoccA * (naoccA + 1) * (naoccA + 2) / 6;
+    // Full set of ijk used here
+    Nijk = naoccA * naoccA * naoccA;
```

Comment:
Is this quantity used anywhere else? You might consider building a new variable to be safe.

Reply:
Only use is the print out for the user, I assume it was simply (wrong) copy & paste from the energy-only subroutine.